src/test/java/com/example/OnlineShopApiApplication/AdminServiceTests.java

```diff
@@ -8,32 +8,30 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 
 import java.math.BigDecimal;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.Mockito.*;
 
 @ExtendWith(MockitoExtension.class)
+@AutoConfigureMockMvc(addFilters = false)
 public class AdminServiceTests {
     private AdminService adminService;
     private EmployeeRepository employeeRepository;
-    private BCryptPasswordEncoder bCryptPasswordEncoder;
 
-    private Employee testEmployee, testEmployee2;
+    private Employee testEmployee;
     private Role testRole;
 
     @BeforeEach
     public void setUp() {
         employeeRepository = mock(EmployeeRepository.class);
```

Comment:
instead of this mock(EmployeeRepository.class); you can use @Mock annotation for the repository:
```
@Mock
private EmployeeRepository employeeRepository;
```